exercise/templates/exercise/_submit_progress.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}
+
+<div class="quiz-submit-overlay submission-overlay hide"></div>
```

Comment:
This element is missing when chapter is leaded with an existing feedback (i.e. first post a solution, reload page, then try to post a solution again). Could this be created in the JS instead?

Reply:
Hmm.. Probably. Is there an example somewhere? 

I mean, I have like 45 minutes left before I need to go (:D) and still some feedback questionnaires to do, I might not be able to get back to this now. But if you have a quick idea, I could try to fix it :)

Reply:
Sounds like a very severe bug. We can't merge this yet. "Somebody" (maybe me) needs to spend some time working on this in the autumn.

Reply:
If the problem is simply that these overlay divs are missing from the page in some situations, then this small template could be always included in exercise.html.

Reply:
or maybe submission_plain.html

Reply:
I added the overlay divs to to submission_plain.html and then I had to fix active elements in chapter.js since the code looks for alert divs in the feedback. The submission_plain.html now always contains a hidden alert div.